### ISSUE TITLE
"Empty repository" banner shown on repo page in between enabling and cloning a repo

### ISSUE THREAD

**Issue by attfarhan** [sourcegraph/sourcegraph#1744]
- **Sourcegraph version:** Insiders

#### Steps to reproduce:

1. Add repositories to a Sourcegraph instance
2. Go to the repositories page
3. Enable a repository
4. Go to the repository
5. Notice it will show "Empty repository" on the page
6. Refresh
7. See that the cloning status appears on the page

![2019-01-10 00 06 00](https://user-images.githubusercontent.com/16265452/50911928-dc349300-146b-11e9-9fd3-1304371d0fe4.gif)

#### Expected behavior:

I expected to immediately be shown the repository cloning progress page, which was the behavior in the past.

#### Actual behavior:

You see "empty repository" and have to refresh to see that the repo is actually already being cloned.

**Comment by nicksnyder** [comment]
I created a separate issue for @felixfbecker's bug because they seem to be different:
https://github.com/sourcegraph/sourcegraph/issues/1751

**Comment by attfarhan** [comment]
@nicksnyder I can't repro this anymore on latest insiders. BUT, I did find another issue while trying to repro this: https://github.com/sourcegraph/sourcegraph/issues/1845.

**Comment by nicksnyder** [comment]
Ok, going to close this

**Comment by felixfbecker** [comment]
I just had this happen again. I added an external service for my local Bitbucket server instance. I then enabled it on the Repositories page:

![image](https://user-images.githubusercontent.com/10532611/55278886-66292500-5312-11e9-9242-cb9d73dad457.png)

Notice how it's not cloned.

Now going to the repository shows:

![image](https://user-images.githubusercontent.com/10532611/55278889-717c5080-5312-11e9-92af-da8c6131ddbe.png)

Which is exactly what is described in this issue.
